backend/src/src-fossology/src/main/java/com/siemens/sw360/fossology/handler/FossologyFileHandler.java

```diff
@@ -28,10 +28,7 @@
 import org.springframework.stereotype.Component;
 
 import java.io.InputStream;
-import java.util.Collection;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```

Comment:
That's a readability optimization that IntelliJ did for me.
`import` is a compile-time instruction. It makes no difference on the compiled byte-code. In particular, it does not affect how many classes are loaded at runtime. You can have your code without imports at all, if you like to spell out fully qualified class names every time, or you can import all the packages with an asterisk import.